tests/libxc/dft-water/input.dat

```diff
@@ -0,0 +1,152 @@
+      'M06':   -76.3513036943,        #TEST
+ 'wB97X-D3':   -76.3631809917,        #TEST
+     'BB1K':   -76.3484183323,        #TEST
+    'B1LYP':    -76.353775513,        #TEST
+'LRC-wPBEh':   -76.3105124439,        #TEST
+     'wB97':   -76.3679448108,        #TEST
+    'B3LYP':   -76.3850899091,        #TEST
+ 'B5050LYP':   -76.3352092048,        #TEST
+   'B1PW91':   -76.3662735666,        #TEST
+   'B97M-V':    -76.359760211,        #TEST
+ 'MGGA_MS2':   -76.3943498117,        #TEST
+ 'MGGA_MS0':   -76.3994320535,        #TEST
+    'wB97X':   -76.3633308351,        #TEST
+  'LC-VV10':   -76.2999698219,        #TEST
+    'B3P86':   -76.4018451142,        #TEST
+  'wB97M-V':   -76.3525086294,        #TEST
+      'M05':   -76.3495839034,        #TEST
+'CAM-B3LYP':   -76.3556131153,        #TEST
+   'M06-2X':    -76.349002993,        #TEST
+  'MPW1B95':   -76.3509384412,        #TEST
+     'TPSS':   -76.3883300339,        #TEST
+    'PWB6K':    -76.409863593,        #TEST
+    'B97-1':   -76.3600126249,        #TEST
+     'EDF2':   -76.3450487085,        #TEST
+    'O3LYP':   -76.3594550251,        #TEST
+   'BHHLYP':   -76.3431790184,        #TEST
+     'VV10':   -76.4786514994,        #TEST
+     'BP86':   -76.3853464707,        #TEST
+      'B97':   -76.3583049672,        #TEST
+    'B97-2':   -76.3592390011,        #TEST
+    'TPSSh':   -76.3806321561,        #TEST
+   'B97-D3':   -76.3448830742,        #TEST
+    'B97-K':   -76.3532885259,        #TEST
+ 'LRC-wPBE':   -76.3231977319,        #TEST
+ 'revTPSSh':    -76.353009954,        #TEST
+      'BOP':   -76.3655205429,        #TEST
+    'B1B95':   -76.3542909285,        #TEST
+  'HCTH147':   -76.3795700801,        #TEST
+    'M11-L':   -76.3634719346,        #TEST
+   'M08-SO':   -76.3328381854,        #TEST
+     'PBE0':   -76.3011288827,        #TEST
+      'PBE':   -76.2983301284,        #TEST
+     'PW91':   -76.3556764834,        #TEST
+    'B97-D':   -76.3448936668,        #TEST
+    'MPW1K':   -76.3571352506,        #TEST
+   'MPWB1K':   -76.3458871109,        #TEST
+  'HCTH407':   -76.3751427842,        #TEST
+     'dlDF':   -76.8428067094,        #TEST
+   'M08-HX':    -76.349820226,        #TEST
+  'HCTH120':   -76.3756781889,        #TEST
+   'M05-2X':   -76.3685940515,        #TEST
+   'B3PW91':   -76.3574933252,        #TEST
+  'wB97X-V':   -76.3540786402,        #TEST
+     'EDF1':   -76.3971372179,        #TEST
+  'revTPSS':   -76.3563235056,        #TEST
+  'wB97X-D':   -76.3625422648,        #TEST
+    'B97-3':    -76.365757473,        #TEST
+ 'MGGA_MS1':   -76.3863611957,        #TEST
+      'M11':   -76.3548246737,        #TEST
+   'PW6B95':   -76.4636801397,        #TEST
+   'B3LYP5':   -76.3479565264,        #TEST
+    'X3LYP':   -76.3550695373,        #TEST
+     'BLYP':   -76.3649513839,        #TEST
+   'HCTH93':   -76.3690959506,        #TEST
+   'M06-HF':   -76.3438580561         #TEST
+#   'PBEOP':   -76.3113060897,        #TEST
```

Comment:
throw in a note about why these commented out – libxc doesn't have them? or psi can't match them?